documentation/puppet-api/v3/environment_modules.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema":     "http://json-schema.org/draft-04/schema#",
+  "title":       "Environment Modules",
+  "description": "Information about the modules in a Puppet code environment",
+  "type":        "object",
+  "properties": {
+    "modules": {
+      "description": "The array of modules which exist in an environment.",
+      "type": "array",
+      "properties": {
+
+      }
```

Comment:
Nah, unfortunately, I did hand create the whole thing for the environment_classes endpoint.  In retrospect, though, I think there was value in writing up what I thought the payload should look like independent from what was actually generated in a json payload - the first time anyway.  I used `JSON::Validator` in the Ruby json-schema gem (https://github.com/ruby-json-schema/json-schema) to validate the schema against the response data from some sample queries that I made to the environment_classes endpoint.  For reference, the JSON schema for that came in originally on #940.